### PR TITLE
Add help buttons to deployment guide

### PR DESCRIPTION
**Pull request by kogre**
To give people a way forward when getting stuck in the deployment guide, added links to post a github issue using a template. We'll have to see how it scales, but for now a good way to figure out the most common issues and improve the documentation.

### Documentation page
image

### Issue template
image

### CODE REVIEW

File: docs/start-deployment.mdx [hunk 1]
```diff
@@ -170,11 +181,11 @@ Finally, in case you want to let people position their inputs on a map, you'll a
 GOOGLE_MAPS_API_KEY=my_google_maps_api_key
 ```
 
-### Other
+### 5.7 Other <DeploymentHelp step={5.7} />
```
Comment (periclesd) [inline docs/start-deployment.mdx]: not sure why this is the odd one out compared to all other 5.x steps, but i tested locally and both work

File: docs/start-deployment.mdx [hunk 2]
```diff
@@ -2,17 +2,28 @@
 title: Deployment
 ---
 
+export const DeploymentHelp = ({ step }) => {
```
Comment (periclesd) [inline docs/start-deployment.mdx]: wondering if we should include the title too, in case the steps ever change, so we have a snapshot

**Review by periclesd — APPROVED** [review]
LGTM, tested locally and works fine

**Comment by periclesd** [comment]
nice initiative!

File: docs/start-deployment.mdx (hunk 2 above)
Comment (kogre) [reply docs/start-deployment.mdx]: Yes, that's a good point. Would also make more sense for externals seeing the issues and greatly improve searchability of past issues. Will adapt!

File: docs/start-deployment.mdx (hunk 1 above)
Comment (kogre) [reply docs/start-deployment.mdx]: Oops, no reason, just forgot. Will change, risks having floating point precision issues otherwise.

**Pull request merged**